development/compiling/compiling_for_linuxbsd.rst

```diff
@@ -79,6 +79,12 @@ Distro-specific one-liners
 |                  |     sudo zypper install scons pkgconfig libX11-devel libXcursor-devel libXrandr-devel libXinerama-devel \ |
 |                  |             libXi-devel Mesa-libGL-devel alsa-devel libpulse-devel libudev-devel libGLU1 yasm             |
 +------------------+-----------------------------------------------------------------------------------------------------------+
+| **NetBSD**       | ::                                                                                                        |
+|                  |                                                                                                           |
+|                  |     pkg_add pkg-config py37-scons yasm                                                                    |
+|                  |                                                                                                           |
+|                  | For audio support, you can optionally install ``alsa-lib`` and/or ``pulseaudio``.                         |
```

Comment:
Note: I'm still building with those installed to confirm if they actually work.

Reply:
PulseAudio seems to work fine. ALSA doesn't work out of the box, so I'll remove the recommendation.